sflvault/api/api_connector.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from sflvault.api.api import APIHandler
+
+
+class APIConnector:
+
+    def __init__(self):
+
+        self.api = APIHandler(os.getenv("VAULT_SERVER"))
+
+    def get_all_instance(self, base_url):
+        return self.api.get(base_url)
```

Comment:
Rather than have `base_url` be a parameter for each function, can we simply reference `self.base_url` and override this in the `__init__` of each of our subclasses?